src/databricks/labs/remorph/reconcile/execute.py

```diff
@@ -73,31 +110,26 @@ def recon(
 
     ws_client: WorkspaceClient = verify_workspace_client(ws)
 
+    source_dialect = get_dialect(reconcile_config.data_source)
+
     # validate the report type
-    report_type = report_type.lower()
+    report_type = reconcile_config.report_type
     logger.info(report_type)
     validate_input(report_type, {"schema", "data", "row", "all"}, "Invalid report type")
 
-    database_config = DatabaseConfig(
-        source_catalog=table_recon.source_catalog,
-        source_schema=table_recon.source_schema,
-        target_catalog=table_recon.target_catalog,
-        target_schema=table_recon.target_schema,
-    )
-
     source, target = initialise_data_source(
         engine=source_dialect,
         spark=spark,
         ws=ws_client,
-        secret_scope=f"remorph_{get_key_form_dialect(source_dialect)}",
+        secret_scope=reconcile_config.secret_scope,
     )
 
     recon_id = str(uuid4())
     # initialise the Reconciliation
     reconciler = Reconciliation(
         source,
         target,
-        database_config,
+        reconcile_config.config,
```

Comment:
Can you rename this to database config everywhere

Reply:
Done